From df1105b186eccf103277d0ccddd4c88e9f9d3c08 Mon Sep 17 00:00:00 2001
From: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Date: Tue, 24 May 2016 12:29:51 +0200
Subject: Fix testsuite failure when sockets is not defined
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

this fixes the UnboundLocalError issue
"UnboundLocalError: local variable 'sockets' referenced before assignment"
Thanks Ondřej Nový for the help
---
 tornado/test/netutil_test.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tornado/test/netutil_test.py b/tornado/test/netutil_test.py
index 78331ce..79f35c0 100644
--- a/tornado/test/netutil_test.py
+++ b/tornado/test/netutil_test.py
@@ -203,6 +203,7 @@ class TestPortAllocation(unittest.TestCase):
 
     @unittest.skipIf(not hasattr(socket, "SO_REUSEPORT"), "SO_REUSEPORT is not supported")
     def test_reuse_port(self):
+        sockets = []
         socket, port = bind_unused_port(reuse_port=True)
         try:
             sockets = bind_sockets(port, 'localhost', reuse_port=True)
